### PR TITLE
Server security audit

### PR DESCRIPTION
**Pull request by thomas-tran**
- Updated DTO
- Updated for security audit changes

### CODE REVIEW

File: src/ProximaX.Sirius.Chain.Sdk/Infrastructure/DTO/AccountIds.cs [hunk 1]
```diff
@@ -1,17 +1,3 @@
-// Copyright 2019 ProximaX
```
Comment (alvin-reyes) [inline src/ProximaX.Sirius.Chain.Sdk/Infrastructure/DTO/AccountIds.cs]: Why was this removed?

Comment (thomas-tran) [reply src/ProximaX.Sirius.Chain.Sdk/Infrastructure/DTO/AccountIds.cs]: I will added it back
